back/lib/tasks/single_use/20240527_migrate_deprecated_about_this_report_widget.rake

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+namespace :single_use do
+  task migrate_deprecated_about_this_report_widget: [:environment] do |_t, _args|
+    # region HELPER METHODS
+
+    # @param [ContentBuilder::Layout] layout
+    def migrate_about_this_report_widgets!(layout)
+      json = layout.craftjs_json
+
+      ContentBuilder::Layout.transaction do
+        layout.dup.tap do |backup|
+          backup.code = "backup/#{layout.code}/#{layout.id}"
+          backup.content_buildable = nil
+        end.save!
+
+        state = ContentBuilder::Craftjs::State.new(json)
+
+        about_nodes = state.nodes_by_resolved_name('AboutReportWidget')
+
+        about_nodes.each do |node_id, about_node|
+          logo_node = create_logo_node!(state)
+          title_node = create_multiloc_node(about_node, 'about-title', state)
+          text_node = create_multiloc_node(about_node, 'about-text', state)
+
+          new_node_ids = [logo_node].compact + [title_node, text_node]
+          state.replace_node(node_id, new_node_ids)
+        end
+
+        layout.update!(craftjs_json: state.json)
+      end
+    end
+
+    # @param [ContentBuilder::Craftjs::State] state
+    # @return [String, nil]
+    def create_logo_node!(state)
+      logo = AppConfiguration.instance.logo.versions[:small]
```

Comment:
In the `ProjectReport`, and generally in the `TenantLogo` component, we use the `medium` version. Don't know the exact dimensions of these versions, but probably if people set `stretch: true`, the `medium` version will look a bit better than the `small` one

Reply:
I tried both options and was a bit unsure. The small size is closer to the current dimensions but the quality isn't great. The medium size looked better in terms of resolution, but it's quite larger than the current logo. Plus, I'm not sure why its resolution would be actually better than the small one.

So, no perfect solution in my opinion, but I don't mind reverting it to medium if you prefer.